portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,44 @@
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div>
+    <h1>Apoorva Dhanala's Portfolio</h1>
+    <img src="/images/me.jpg" align="right" alt="Here's a picture of me!" width="200" height="300">
+    </div>
+    <p>Hi! My name is Apoorva Dhanala and I am a rising junior at the University of Georgia studying Computer Science 
+      with an emphasis in Artificial Intelligence. At UGA, I am founder and president of UGA’s chapter of Girls Who Code 
+      where we work with local, underprivileged Athens-Clarke County schools where UGA students volunteer weekly hoping to 
+      expose students to coding while closing the gender gap in technology. I am also involved with the Georgia Recruitment 
+      Team where I represent the admissions office as an ambassador at panels and tours, Athens Prison Tutorial where I tutor 
+      prison inmates at the Athens-Clarke County Jail who lack their high school diplomas and help them improve their skills 
+      in order to pass their GED examination, and with the Honors Program Student Council where I work to help bring academic 
+      and social opportunities to students in the Honors Program. Outside of classes and club involvement, I am a CURO Honors 
+      Research scholar where I have been involved with Linguistics and Natural Language Processing research and most recently, 
+      Android App development work.  
+    </p>
+    <p>
+    I was born and raised in Atlanta, Georgia and have a little sister. I have learned piano and the classical Indian dance 
+      form, Kuchipudi, since I was 5 years old; the skills I have learnt from these art forms, from patience and perseverance 
+      to responsibility and delicateness, are a big part of who I am today. Some of my favorite things to do are travel, take 
+      pictures, cook, and watch TV. I love traveling to new places and getting to see different parts of the world and I also 
+      love photography and taking pictures whether they are travel pictures or senior pictures. As for cooking and baking, some
+      of my favorite things to experiment with are different types of pasta & sauces and also chocolate chip cookies! Whenever 
+      I watch TV, some of my go-to’s are Suits, Modern Family or Scandal. 
+    </p>
+    <p>
+    Click on any of the links below to find out more about who I am, whether it has to do with my professional experiences or 
+      more about my interests and hobbies! 
+    </p>
+    
+    <p>Click to see one of my favorite quotes!</p>
+    <button onclick="addRandomQuote()">Let's see it!</button>
+    <div id="quote-container"></div>
+
+    <p><a href="resume.html">My Resume</a></p>
+    <p><a href="photography.html">My Photography</a></p>
+    <p><a href="favorites.html">Favorite Songs, TV shows, and more!</a></p>
```

Comment:
This page isn't present in this pull request, so remove it now and add it back in a subsequent one?

Reply:
I'm not sure what you mean by remove it as I don't think I committed it with anything - I did git log follow on the index.html file and the last time I committed it was when I changed formatting of my background content info that Jenny asked for which was around 11:14 this morning. I don't think I'm understanding what you're asking correctly - could you explain/guide me towards what git commands I need to use to fix this? Thanks!

Reply:
I meant to remove this particular line from your file because I don't see the file 'favorites.html' in this PR.